mods/ra2/rules/soviet-infantry.yaml

```diff
@@ -92,12 +92,12 @@ shk:
 	Inherits: ^Infantry
 	Buildable:
 		Queue: Infantry
-		BuildAtProductionType: Infantry
```

Comment:
This exists so you can build them at the `nahand` or `gapile`, whichever is free.